pkg/api/client/client_public.go

```diff
@@ -33,6 +33,20 @@ type publicAPI interface {
 	NewReverseListener(token string) (*revdial.Listener, error)
 	AuthPublicKey(req *models.PublicKeyAuthRequest, token string) (*models.PublicKeyAuthResponse, error)
 	ReportUsage(ur *models.UsageRecord, billingURL string) (int, error)
+	ReportDelete(ns *models.Namespace, billingURL string) (int, error)
+}
+
+func (c *client) ReportDelete(ns *models.Namespace, billingURL string) (int, error) {
+	res, _, errs := c.http.Delete(fmt.Sprintf("http://%s:8080/api/billing/namespace-subscription", billingURL)).Send(struct {
```

Comment:
This fixed URL is likely to cause errors. We need to use this as a global setting.